OutOfSchool/OutOfSchool.BusinessLogic/Services/ApplicationService.cs

```diff
@@ -584,8 +584,8 @@ private static Expression<Func<Application, bool>> PredicateBuild(
             var tempPredicate = PredicateBuilder.False<Application>();
             tempPredicate = tempPredicate
                 .Or(a => a.Workshop.Title == filter.SearchString)
-                .Or(a => a.Workshop.ProviderTitle.StartsWith(filter.SearchString, StringComparison.InvariantCultureIgnoreCase))
-                .Or(a => a.Workshop.ProviderTitleEn.StartsWith(filter.SearchString, StringComparison.InvariantCultureIgnoreCase))
+                .Or(a => a.Workshop.Provider.FullTitle.StartsWith(filter.SearchString, StringComparison.InvariantCultureIgnoreCase))
```

Comment:
The provider is not included in the `includeFunc` property. As it is used for a predicate - it will eagerly join it anyway. But double check that the provider title is not accessed anywhere in `ApplicationService` via mapping. As it will lead to lazy loading n+1 :) or just add provider to `includeFunc` as nested include for workshop